plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryStorageArrowPageSource.java

```diff
@@ -52,14 +55,17 @@ public class BigQueryStorageArrowPageSource
     private final AtomicLong readBytes = new AtomicLong();
     private final BigQueryReadClient bigQueryReadClient;
     private final BigQuerySplit split;
```

Comment:
Unrelated to current PR, but please improve this anyway.
We shouldn't be holding on to `BigQuerySplit`, we just use `io.trino.plugin.bigquery.BigQuerySplit#getStreamName` from it which can be kept as a member variable instead of the whole split.

Reply:
Done!